docs/USAGE.md

```diff
@@ -2,37 +2,95 @@
+
+An image can be loaded into the `LocalFour` space using the OVE APIs, by running the following commands using [curl](https://curl.haxx.se/docs/manpage.html).
+
+Linux/Mac:
+
+```sh
+curl --header "Content-Type: application/json" --request DELETE http://OVE_CORE_HOST:PORT/section
+curl --header "Content-Type: application/json" --request POST --data '{"app": {"url": "http://OVE_APP_IMAGES_HOST:PORT","states": {"load": {"tileSources": "https://openseadragon.github.io/example-images/highsmith/highsmith.dzi"}}}, "space": "LocalFour", "h": 500, "w": 500, "y": 0, "x": 0}' http://OVE_CORE_HOST:PORT/section
+```
 
-It is recommended to use Chrome as the browser, because that is what is primarily used for development, testing, and production at the [DSI](https://www.imperial.ac.uk/data-science/).
+Windows:
 
-In a larger OVE installation opening browser windows may be sufficiently time-consuming for automation to be worthwhile.
-Within the [DSI](https://www.imperial.ac.uk/data-science/), we use PowerShell scripts to launch browsers, make them full-screen, and navigate them to the correct URL.
-On macOS, AppleScript could be used instead.
+```sh
+curl --header "Content-Type: application/json" --request DELETE http://OVE_CORE_HOST:PORT/section
+curl --header "Content-Type: application/json" --request POST --data "{\"app\": {\"url\": \"http://OVE_APP_IMAGES_HOST:PORT\", \"states\": {\"load\": {\"tileSources\": \"https://openseadragon.github.io/example-images/highsmith/highsmith.dzi\"}}}, \"space\": \"LocalFour\", \"h\": 500, \"w\": 500, \"y\": 0, \"x\": 0}" http://OVE_CORE_HOST:PORT/section
+```
 
-## Creating sections and launching apps
+These commands clear all sections on OVE, and creates a new section with an instance of the [Image App](https://github.com/ove/ove-apps/tree/master/packages/ove-app-images) running within it. In this example we are loading an image with a [Deep Zoom](https://docs.microsoft.com/en-us/previous-versions/windows/silverlight/dotnet-windows-silverlight/cc645077(v=vs.95)) file format.
 
-To actually display anything within OVE, it is necessary to create a section, load an app into it, and potentially also set the state of this app.
+## Controlling OVE Apps and designing interactive visualisations
 
-This can be done in several ways:
+Once an App has been loaded into an OVE `space` it can be controlled using the corresponding controller, that provides app-specific functionality. The controller of the [Image App](https://github.com/ove/ove-apps/tree/master/packages/ove-app-images) supports panning and zooming of images that have been loaded. The controller can be loaded into a separate browser window by accessing the URL `http://OVE_APP_IMAGES_HOST:PORT/control.html?oveSectionId=0`.
```

Comment:
```suggestion
Once an App has been loaded into an OVE `space` it can be controlled using the corresponding controller, which provides app-specific functionality. For example, the controller of the [Image App](https://github.com/ove/ove-apps/tree/master/packages/ove-app-images) supports panning and zooming of images that have been loaded. The controller can be loaded into a separate browser window by accessing the URL `http://OVE_APP_IMAGES_HOST:PORT/control.html?oveSectionId=0`.
```